Lombiq.HelpfulLibraries.AspNetCore/Docs/Security.md

```diff
@@ -0,0 +1,8 @@
+# Lombiq Helpful Libraries - ASP.NET Core Libraries - Security
```

Comment:
Link this to https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI/Docs/SecurityScanning.md and vice versa.